### ISSUE TITLE
Custom view, call parent callback?

### ISSUE THREAD

**Issue by musicist288** [gitana/alpaca#255]
I'm creating a custom view for my form controls as a child of boostrap-edit. I'd like to call the parent view's `control` callback before performing some additional logic, but can't figure out a way to do it from the child other than: `Aplaca.views['bootstrap-edit'].callbacks.control.call(this)`

Is there a better way? After looking at the `registerView` method, it doesn't look like it.

**Comment by uzquiano** [comment]
Yes, the views require you to manage your own closures.  They're not object oriented in the same way as the fields framework.  They don't maintain a parent "this" or anything like that within the callback scope.  For callbacks, we actually set "this" in most cases to be the underlying control instance.

So, yes, both good and bad depending on your love for JavaScript.  The method you suggest would work just fine in practice.

If you or anyone else in the community has a suggestion for a convenience pattern here, it'd be good to shoot around some ideas.  Is something beyond the static accessor more helpful?
